apps/src/aichat/views/modelCustomization/ModelDescriptionPanel.tsx

```diff
@@ -0,0 +1,106 @@
+import React, {useEffect, useState, useRef} from 'react';
+
+import SimpleDropdown from '@cdo/apps/componentLibrary/dropdown/simpleDropdown';
+import {BodyThreeText, StrongText} from '@cdo/apps/componentLibrary/typography';
+import Button from '@cdo/apps/componentLibrary/button/Button';
+
+import styles from './compare-models-dialog.module.scss';
+
+type ModelDescription = {
```

Comment:
Should we move this to `types.ts`?

Reply:
I think when we've chatted about this before we've been ok leaving types used in one place in the file that they're used?